index.html

```diff
@@ -2,10 +2,43 @@
 <html>
   <head>
     <title>Relaxr</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600,800" rel="stylesheet" type="text/css">
     <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
   <body>
     <header>
+    	<h1>Relaxr</h1>
+    	<center>
```

Comment:
Nice job finding this tag. It's crazy old though and unfortunately HTML5 does not support it. But I applaud that you found it! Use css to center stuff instead 😉